charts/linkerd-control-plane/templates/destination.yaml

```diff
@@ -194,6 +194,8 @@ spec:
       {{- $_ := set $tree.Values.proxy "await" $tree.Values.proxy.await }}
       {{- $_ := set $tree.Values.proxy "loadTrustBundleFromConfigMap" true }}
       {{- $_ := set $tree.Values.proxy "podInboundPorts" "8086,8090,8443,9443,9990,9996,9997" }}
+      {{- $_ := set $tree.Values.proxy "outboundDiscoveryCacheUnusedTimeout" "" }}
```

Comment:
Is the purpose of this to unset the value to ensure that the control plane proxies use the default value for these cache timeouts?  Should we just explicitly set them here to the values we want?

Reply:
Yeah, we unset them so they receive the defaults. We can hardcode it to the default value instead so it's a bit more visible.